Source/diablo.cpp

```diff
@@ -2237,24 +2254,25 @@ void initKeymapActions()
 				    UseInvItem(myplr, INVITEM_BELT_FIRST + i);
 			    }
 		    },
+		    [&]() { return !isPlayerDead(); },
 		});
 	}
 	keymapper.addAction({
 	    "QuickSave",
 	    DVL_VK_F2,
 	    [] { gamemenu_save_game(false); },
+	    [&]() { return !gbIsMultiplayer && !isPlayerDead(); },
 	});
 	keymapper.addAction({
 	    "QuickLoad",
 	    DVL_VK_F3,
 	    [] { gamemenu_load_game(false); },
-	    Keymapper::Action::IfDead::Allow,
+	    [&]() { return !gbIsMultiplayer && gbValidSaveFile; },

```

Comment:
Technically the check for a save state should be good enough, but probably best to keep both :)